docs/api/index.md

```diff
@@ -23,7 +25,7 @@ The API documentation assumes you are familiar with REST APIs. If you're not, th
 * [KnowledgeOwl's guide to working with APIs](https://support.knowledgeowl.com/help/working-with-apis){:target=_blank .external-link}: a basic introduction, including examples of how to call REST APIs.
 * [IBM Cloud Learn Hub - What is an Application Programming Interface (API)](https://www.ibm.com/cloud/learn/api){:target=_blank .external-link}: this gives a general, but technical, introduction to APIs.
 * [IBM Cloud Learn Hub - What is a REST API?](https://www.ibm.com/cloud/learn/rest-apis){:target=_blank .external-link}: more detailed information about REST APIs.
-* [MDN web docs - An overview of HTTP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview){:target=_blank .external-link}: REST APIs work over HTTP, and use HTTP verbs, or methods, to indicate the type of action to perform.
+* [MDN web docs - An overview of HTTP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview){:target=_blank .external-link}: REST APIs work over HTTP and use HTTP verbs, or methods, to identify the action to perform.
```

Comment:
No strong feelings, but maybe "specify" is a bit more active if it doesn't run afoul of any style rules.

Reply:
Good call. "Specify" is allowed. :)